css/main.scss

```diff
@@ -119,6 +119,10 @@ a {
 	transition: color 0.2s;
 }
 
+a.call-to-action {
```

Comment:
Can add `.mt-3` class or something to the button to avoid custom css. We could have different margins for different call to actions buttons.